lte/gateway/python/integ_tests/federated_tests/fabfile.py

```diff
@@ -132,19 +118,6 @@ def configure_orc8r(on_vagrant='False'):
             run(command_feg)
 
 
-def clear_gateways():
```

Comment:
This function was actually quite helpful in testing during the fabric upgrade, because it lets you easily deregister the FEG and AGW. Probably similar to all the `stop_{service}` functions, they're not used anywhere, but they are convenient functions